ktor-server/ktor-server-core/jvm/src/io/ktor/features/DataConversion.kt

```diff
@@ -68,7 +68,7 @@ public class DataConversion(private val converters: Map<Type, ConversionService>
             return DataConversion(configuration.converters)
```

Comment:
`ConversionService` provides function `convert` taking `ConversionService` so you need to add functions with the new type as well

Reply:
I didn't deprecate his class yet, because it's coupled with the Locations feature. I will address it in the next PRs to not overload this one with changes.